src/patternfly/utilities/BackgroundColor/BackgroundColor.scss

```diff
@@ -0,0 +1,12 @@
+// stylelint-disable
+
+// Background color options
+$pf-u-background-color-options: (
+  /* Background color */
+  background-color-100:  (background-color var(--pf-global--BackgroundColor--100)),
+  background-color-200:  (background-color var(--pf-global--BackgroundColor--200)),
+);
```

Comment:
I think we should add these colors, as defined by design, to utility background colors:

![Screen Shot 2020-09-16 at 3 48 58 PM](https://user-images.githubusercontent.com/5385435/93385367-2cb0e600-f834-11ea-8083-0bc86496419c.png)